### ISSUE TITLE
ContentRenderer doesn't work

### ISSUE THREAD

**Issue by buditanrim** [nuxt/content#2477]
Hello. everyone. I'm a bit confused when following the documentation

Given I have a markdown file:
```
 /content/article/design.md
---
title: How to design
year: 2023
---
The content of this markdown
```

```
In my /pages/article/[...slug].vue 

<template>
  <main>
    <ContentDoc>
      <template #default="{ doc }">
        <p>{{ doc.title }}</p>
        <p>{{ doc.year }}</p>
        <ContentRenderer :value="doc"/>
      </template>
    </ContentDoc>
  </main>
</template>
```
Problem: The title and year is rendered. But the <ContentRenderer> doesn't return anything.
Question: What component can I use to render the content from the markdown?

**Comment by AssetCEO** [comment]
In /pages/article/[...slug].vue 

```
    <ContentDoc v-slot="{ doc }">
      <article>
        <h1>{{ doc.title }}</h1>
        <p>{{ doc.year }}</p>
        <ContentRenderer :value="doc" />
      </article>
    </ContentDoc>
```

**Comment by buditanrim** [comment]
Hello, I found that `NuxtLayout` create this issue, I'll provide more information soon.

**Comment by code4reason** [comment]
I experienced a renderproblem with links in Markdown files in a Nuxt 3 app with Nuxt Content v2. They rendered as plain text and was not clickable. Tried different browsers, incognito, deleted node cache, node_modules, .nuxt. Then i found this issue and removed NuxtLayout from my app.vue file. Then my links in markdown files worked. No solution, but a similar problem i guess? BTW, i upgraded NUXT, Vue, and Nuxt Content to latest versions.
